platform/i18n/src/index.js

```diff
@@ -42,6 +42,7 @@ const locizeOptions = {
 
 const envUseLocize = !!config.USE_LOCIZE;
 const envApiKeyAvailable = !!config.LOCIZE_API_KEY;
+const DEFAULT_LANGUAGE = 'en-US';
```

Comment:
This should probably be exposed in viewer config? (potentially worth creating a follow-up issue)